src/SprykerEco/Service/ComputopApi/ComputopApiService.php

```diff
@@ -108,7 +98,7 @@ public function decryptResponseHeader(array $responseHeader, $password): array
         $responseDecryptedString = $this->getBlowfishDecryptedValue(
             $responseHeader[ComputopApiConfig::DATA],
             $responseHeader[ComputopApiConfig::LENGTH],
-            $password
+            $password,
```

Comment:
this method should be also updated
service should only contain factory/model call ant not business logic

Reply:
Fixed

Reply:
Fixed